src/Services/DatabaseBackup/AbstractDatabaseBackup.php

```diff
@@ -33,18 +33,23 @@
      *
      * @var array
      */
-    protected $classNames;
+    protected $classNames = [];
 
     public function __construct(ContainerInterface $container, FixturesLoaderFactory $fixturesLoaderFactory)
     {
         $this->container = $container;
         $this->fixturesLoaderFactory = $fixturesLoaderFactory;
     }
 
-    public function init(array $metadatas, array $classNames): void
+    public function init(array $metadatas, array $classNames, bool $append = false): void
     {
         $this->metadatas = $metadatas;
-        $this->classNames = $classNames;
+
+        if ($append) {
+            $this->classNames = array_merge($this->classNames, $classNames);
```

Comment:
When appending fixtures, keep track of `classNames` which had been already loaded.